packages/plugins/plugins/plugin-next-auth/src/auth/providers.js

```diff
@@ -17,6 +17,7 @@
 // This syntax does not work because next-auth is not an es module.
 // export { default as Auth0Provider } from 'next-auth/providers/auth0';
 
+import OpenIDConnectProvider from './OpenIDConnectProvider.js';
```

Comment:
Please move this to the bottom of imports, as it is convention to first import from external libs.

Reply:
Change made in 12a5834bba08ca463f157900825fa039075dd7d0